src/Console/ConsoleOptionParser.php

```diff
@@ -576,6 +576,7 @@ public function addSubcommand($name, array $options = [])
             $command = $name;
             $name = $command->name();
         } else {
+            $name = Inflector::underscore($name);
```

Comment:
Yeah, that was the concern I had above. I think I have an idea on how to keep things here rather BC.